packages/schematics/angular/ng-new/index.ts

```diff
@@ -60,11 +61,16 @@ export default function (options: NgNewOptions): Rule {
     zoneless: options.zoneless,
   };
 
+  const configOptions: AiConfigOptions | false =
+    options.aiConfig?.includes(AiConfig.None) !== true &&
+    (options.aiConfig as unknown as AiConfigOptions);
+
   return chain([
     mergeWith(
       apply(empty(), [
         schematic('workspace', workspaceOptions),
         options.createApplication ? schematic('application', applicationOptions) : noop,
+        configOptions ? schematic('ai-config', configOptions) : noop,
```

Comment:
Reduce duplicate logic, move the none to the ai-config schematic and always call the schematic

```suggestion
  return chain([
    mergeWith(
      apply(empty(), [
        schematic('workspace', workspaceOptions),
        options.createApplication ? schematic('application', applicationOptions) : noop,
        schematic('ai-config', { tool: options.aiConfig }),
```